data-serving/data-service/schemas/cases.indexes.json

```diff
@@ -13,7 +13,8 @@
             "location.place": "text",
             "location.name": "text",
             "pathogen.name": "text",
-            "caseReference.sourceUrl": "text"
+            "caseReference.sourceUrl": "text",
+            "caseReference.verificationStatus": "text"
```

Comment:
this is for full text search, you'll want to add an index below for this field to make it searchable using a keyword

Reply:
Oh got it, thanks. I was copying sourceUrl which didn't have one, so I added a keyword index for both.